site/content/en/docs/Reference/API/Kustomization File/configMapGenerator.md

```diff
@@ -6,3 +6,81 @@ weight: 6
 description: >
     Generate ConfigMap resources.
 ---
+
+## configMapGenerator
+
+ConfigMap objects can be generated by adding a `configMapGenerator` entry to the `kustomization.yaml` file.
+
+See [Generating ConfigMaps Tasks] for examples of how to use `configMapGenerator`.
+
+---
+
+* **apiVersion**: kustomize.config.k8s.io/v1beta1
```

Comment:
We probably want to move `kind: Kustomization` into its own Reference Kustomization file, just so we don't repeat ourselves or overwhelm the reader with multiple contexts. In that file, we can include all the supported fields and point them to the respective field Reference pages, like this one.

Reply:
I like this idea! I added a new kustomization.md file.